styles.css

```diff
@@ -0,0 +1,1095 @@
+* {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Lato', sans-serif;
+}
+
+body {
+  width: 100%;
+  min-height: 100%;
+  contain: content;
+}
+
+@media only screen and (max-width: 768px) {
+  .head {
+    display: none;
+  }
+
+  .hamburger {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    gap: 2px;
```

Comment:
- [ ] As suggested by the previous reviewer,  Please consider using `min-width` media queries as you're working on a **mobile-first approach**, using both max-width and min-width is redundant. With min-width media queries, you explicitly tell the browser to apply these styles to devices **with a minimum width of the value you defined**, in this case, 768px. So your mobile styles can go outside the media query because those devices have less width. Please consider fixing this! 🤓 🛠️